### ISSUE TITLE
Bitmap back to file?

### ISSUE THREAD

**Issue by jeremyworger** [m4tx/txttobitmap#1]
Hey there, have you considered adding the inverse solution for your project, to convert the bitmap back to the original file?
Just curious. 
Cheers,

J

**Comment by m4tx** [comment]
Yes, that would make sense, for example for building a simple steganography tool. However, so far I have lacked motivation to do so :) If you would like to create a PR for that, I will be more than happy to review and merge it.
